### PR TITLE
Configure to deploy to heroku

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

To build and deploy to heroku:

```sh
# Create app in heroku.
heroku login
heroku create
heroku config:set NODE_HOST=(heroku url from the previous commands output)


# Deploy app.
# You can be on any branch and run this exact command to deploy your current local commit to heroku
git push -f heroku HEAD:master
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -15178,7 +15152,7 @@
       "dev": true
     },
     "node_modules/node-zendesk": {
-      "resolved": "git+ssh://git@github.com/jfrerich/node-zendesk.git#c5909d6e6727fd6321e267454ff239a46d026808",
+      "resolved": "git+https://github.com/jfrerich/node-zendesk.git#c5909d6e6727fd6321e267454ff239a46d026808",
```
Comment (mickmister) [inline package-lock.json:15155]: I had to change these from `ssh` to `https` for heroku to be able to fetch them

**Comment by jfrerich** [comment]
Thanks, @mickmister! Could you add this to the README?  I've never actually deployed to heroku myself.

**Comment by mickmister** [comment]
> Thanks, @mickmister! Could you add this to the README? I've never actually deployed to heroku myself.

@jfrerich Done :+1:

**Review by jfrerich — APPROVED** [review]
Thanks!

**Review by hanzei — CHANGES_REQUESTED** [review]
Hedoku uses node 18 by default, which fails to install dependencies. I can reproduce the `npm i` issues on my local machine when using node 16:

```
 npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: ts-jest@27.0.5
npm ERR! Found: babel-jest@24.9.0
npm ERR! node_modules/babel-jest
npm ERR!   dev babel-jest@"24.9.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional babel-jest@">=27.0.0 <28" from ts-jest@27.0.5
npm ERR! node_modules/ts-jest
npm ERR!   dev ts-jest@"27.0.5" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: babel-jest@27.5.1
npm ERR! node_modules/babel-jest
npm ERR!   peerOptional babel-jest@">=27.0.0 <28" from ts-jest@27.0.5
npm ERR!   node_modules/ts-jest
npm ERR!     dev ts-jest@"27.0.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

I see two options here: Fix the dependency issue or define a node version using the steps from https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version

**Pull request closed without merging**